src/actions/admin.js

```diff
@@ -118,7 +122,7 @@ export function getComplianceReport(reportId) {
     );
 }
 
-export function getComplianceReports(page = 0, perPage = General.PER_PAGE_DEFAULT) {
+export function getComplianceReports(page: number = 0, perPage: number = General.PAGE_SIZE_DEFAULT): ActionFunc {
```

Comment:
I think this is a small bug detected by flow (General.PER_PAGE_DEFAULT is undefined), the General.PAGE_SIZE_DEFAULT